Make make_exe.sh executable

fnogatz opened the pull request.

Kaljurand commented:
Thanks for the pull request, but I think the now executable file should start with a shebang line to make sure that it is executed by /bin/sh.

In general, we should probably simplify the build script not to look for the now obsolete "pl" executable.

fnogatz commented:
> but I think the now executable file should start with a shebang line to make sure that it is executed by /bin/sh.

Good point, I added the shebang and removed the DOS/Windows style line endings (CR LF).
